src/engine/enginebuffer.cpp

```diff
@@ -575,6 +576,12 @@ void EngineBuffer::ejectTrack() {
 
 void EngineBuffer::slotPassthroughChanged(double enabled) {
     if (enabled) {
+        // If no vinyl input is configured, do nothing
```

Comment:
We have here the issue that "passthrough" is used in other places to check if it is active. 
When ignoring it here, we get an ambiguous states. 
I see two possible solutions:
* Don't rely on "pathtrough" eselwhere.
* Use connect valueChangeRequest and reject passthrough if it is not configured.

Reply:
> Use connect valueChangeRequest and reject passthrough if it is not configured.

this feels right to me -- rejecting the change would cause the pass button to turn off, correctly. the user may wonder why passthrough isn't turning on, but that's more likely to send them to the sound preferences than it turning on and just not working